lib/auth/MyPadsAuthProvider.js

```diff
@@ -4,14 +4,33 @@ import getConfig from 'next/config';
 // @TODO: If a user logs in with a different device, the saved auth token becomes invalid and they need to login again
 
 class MyPadsAuthProvider {
-    constructor(configuration) {
+    constructor(configuration, token) {
```

Comment:
Is this new parameter ever being used? If not yet, can we remove it for now?

Reply:
ok, is deleted.